flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/SchedulingExecutionVertex.java

```diff
@@ -27,7 +27,7 @@
 /**
  * Scheduling representation of {@link ExecutionVertex}.
  */
-public interface SchedulingVertex {
+public interface SchedulingExecutionVertex {
```

Comment:
Why did you rename this interface?

Reply:
https://github.com/apache/flink/pull/8233#discussion_r277975059

Reply:
Yes, Gray suggested changing it to SchedulingExecutionVertex. I think both works. Which do you prefer?